packages/ui-components/src/components/windowedlist.ts

```diff
@@ -1182,18 +1270,56 @@ export class WindowedList<
     }
   }
 
-  protected _viewModel: T;
-  private _innerElement: HTMLDivElement;
+  /**
+   * Render virtual scrollbar.
+   */
+  private _renderScrollbar(): void {
+    const { node, renderer, viewModel } = this;
+    const content = node.querySelector('.jp-WindowedPanel-scrollbar-content')!;
+
+    while (content.firstChild) {
+      content.removeChild(content.firstChild);
+    }
+
+    const list = viewModel.itemsList;
+    if (list?.length) {
+      for (let index = 0; index < list.length; index += 1) {
+        const item = renderer.createScrollbarItem(index, list.get(index), this);
+        item.dataset.index = `${index}`;
+        content.appendChild(item);
+      }
+    }
+  }
+
+  /**
+   * Handle `pointerdown` events on the virtual scrollbar.
+   */
+  private _evtPointerDown(event: PointerEvent): void {
```

Comment:
Should it be on PointerUp instead?

If the user press than move while pressing, what is expected to happen?